app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -83,13 +86,13 @@ class WeightedQueryStrategy
         daily_decay_factor: {
           clause: "(current_date - articles.published_at::date)",
           cases: [
-            [0, 1], [1, 0.95], [2, 0.9],
-            [3, 0.85], [4, 0.8], [5, 0.75],
-            [6, 0.7], [7, 0.65], [8, 0.6],
-            [9, 0.55], [10, 0.5], [11, 0.4],
-            [12, 0.3], [13, 0.2], [14, 0.1]
+            [0, 1], [1, 0.99], [2, 0.985],
```

Comment:
Reading note: This is a large decrease in the decay rate across all days (recency weighting was weakened).

I feel like that wasn't called out clearly in the PR description. This might be "removed a few levers that weren't doing much", but it looks like a large part of the initial experiment was recency weighting and that's now much weaker than the first time.